bin/create

```diff
@@ -14,6 +14,16 @@ if [ $(id -u) -ne 0 ]; then
     exit 1
 fi
 
+# FIXME: We really should ensure this also works on
+#        classic, its an easy way to get a chroot there.
+#
+#        However as it is written currently it will break
+#        on classic so disable explicitly to avoid confusion.
+if ! grep -q 'ID=ubuntu-core' /etc/os-release; then
```

Comment:
This is fine.

Reply:
Woah, actually, is it? I must be wrong in one of the places I commented about.

Reply:
I think this needs to look at `/var/lib/snapd/hostfs/etc/os-release (and look at the symlink value as well it may be a relative symlink). This is always true. Feels untested.

Reply:
It does - and yes, this snap is in a bad state, it needs tests and it needs shellcheck. I'm working on it :)